### PR TITLE
Converted (bilibili.md) from English (CN) to English (Intl') (Probably) (Most likely) (Kinda) (Maybe)

### PR DESCRIPTION
**Pull request by cydia2020**

### CODE REVIEW

**Comment by Integ** [comment]
Amazing! How can you dig out that the word `mpw` is a typo of `now`.
Definitely, the Bilibili Security Group needs a grammar checker and a writing assistant like you.

**Comment by sgkoishi** [comment]
Minor grammar stuff:
`We have read and understand` -> understood
`Many people has forked it` -> have

**Comment by cydia2020** [comment]
Yeh cheers for pointing these out, they had gone unnoticed since most people here are probably for the jokes (on Bilibili). No need to waste time on it.😉

But they have been fixed now. Good on ya.

**Comment by C8H17OH** [comment]
How to say...As a Chinese programmer and a Bilibili user, I feel ashamed of such a Chinese high tech (><) Who can imagine that it is a Nasdaq company (><)

**Comment by gonemad4u** [comment]
lmao bilibili is such a joke

**Comment by xinhuang327** [comment]
lao mao!

**Comment by Eric000000** [comment]
Use Google Translate before sending this, please.
You can avoid a lot of mistakes.

**Comment by cydia2020** [comment]
Folks, ftp://ddns.cc27.xyz/bilibili-source-code.zip
Cough, cough, lol

File: 2019/04/2019-04-23-bilibili.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
 
-We have read and understand GitHub's Guide to Filling a DMCA Notice
+We have read and understood GitHub's Guidelines of filing a DMCA Notice.
 
-Some one created a Repository called 'openbilibili/go-common' two hours ago and many people forked it or clone it and reupload the repository,which contained too much sensitive information,such as secret key,database address,very important api key and so on.If these information are found by other people in Github,our company will be left out of pocket.
+Someone created a repository called 'openbilibili/go-common' two hours ago. Many people have forked, cloned, and reupload the repository, which contained a lot of sensitive information, such as secret keys, database addresses, very important API keys, et cetera. If this information is leaked, our company will be left out of business.
 
-The following is Github's URL and Screenshots about this  Repository:
+The following is GitHub’s URL and screenshot about the aforementioned repository:
```
Comment (starkwang) [inline 2019/04/2019-04-23-bilibili.md]: `GitHub’s` -> `GitHub's`

Comment (cydia2020) [reply 2019/04/2019-04-23-bilibili.md]: A normal day with Japanese IME

Comment (cydia2020) [reply 2019/04/2019-04-23-bilibili.md]: Fixed

**Comment by jessephus** [comment]
😂

**Comment by cydia2020** [comment]
@jessephus '*trumpface*' Sad

File: 2019/04/2019-04-23-bilibili.md [hunk 2]
```diff
@@ -22,9 +24,10 @@ The following is our contact information:
 -   Company Address: [private]   
 -   Website: https://www.bilibili.com/
 
-I have a good faith belief that use of the copyrighted materials described above on the infringing web pages is not authorized by the copyright owner, or its agent, or the law. I have taken fair use into consideration.
+I am in good faith believing that the use and distribution of aforementioned copyrighted materials on the infringing repositories were not authorized by the copyright owner, its agent, or the law. I have taken fair use into consideration.
```
Comment (chenyiping1995) [inline 2019/04/2019-04-23-bilibili.md:27]: I believe this sentence has no problem whatsoever, as this is the exact copy from the [*Guide to Submitting a DMCA Takedown Notice*](https://help.github.com/en/articles/guide-to-submitting-a-dmca-takedown-notice).

> Include the following statement: "I have a good faith belief that use of the copyrighted materials described above on the infringing web pages is not authorized by the copyright owner, or its agent, or the law. I have taken fair use into consideration."

> Also include the following statement: "I swear, under penalty of perjury, that the information in this notification is accurate and that I am the copyright owner, or am authorized to act on behalf of the owner, of an exclusive right that is allegedly infringed."

Comment (cydia2020) [reply 2019/04/2019-04-23-bilibili.md]: Yeh nah mate I don't really care😂

**Comment by hauter** [comment]
> Minor grammar stuff:
> `We have read and understand` -> understood
> `Many people has forked it` -> have

"I have read and understand GitHub's Guide " is an exact copy from the guide of GitHub notice. 

 [guide-to-submitting-a-dmca-takedown-notice](https://help.github.com/en/articles/guide-to-submitting-a-dmca-takedown-notice)

**Comment by hauter** [comment]
It's absolutely difficult for a Chinese developer to write perfect English notice, and I believe that most of the users posted comments here are from China. LOL.

**Comment by C8H17OH** [comment]
> It's absolutely difficult for a Chinese developer to write perfect English notice, and I believe that most of the users posted comments here are from China. LOL.

Yeah, because it is quite a piece of news in China, especially among IT practitioners😂. And I still can't understand why Bilibili does not just fill the form GitHub provide like other corporations who have sent DMCA notices, but try to write a letter that comes out like a junior high student's composition (捂脸).

**Comment by cydia2020** [comment]
@C8H17OH Agreed, at least put some efforts in.

**Pull request closed without merging**